src/commands/install.yml

```diff
@@ -21,20 +20,18 @@ parameters:
     default: amd64
 steps:
   - run:
-      name: Get download URL
+      name: Download and set up executable
       command: |
-        url="https://bin.equinox.io/c"
-        file="rainforest-cli"
-        case << parameters.channel >> in
-          stable) url="$url/htRtQZagtfg"; file="$file-stable";;
-          beta) url="$url/csTtrGrMBYr"; file="$file-beta";;
-          dev) url="$url/6dz6cwxHnZ8"; file="$file-dev";;
-          *) echo "Invalid channel << parameters.channel >>" 1>&2; exit 1;;
-        esac
+        release_url=https://api.github.com/repos/rainforestapp/rainforest-cli/releases
+        if [ -n "<< parameters.version >>" ]; then
+          release_url="$release_url/tags/<< parameters.version >>"
+        else
+          release_url="$release_url/latest"
+        fi
+        echo $release_url
 
-        file="$file-<< parameters.platform >>-<< parameters.architecture >>.tgz"
-        url="$url/$file"
+        asset_url=$(curl $release_url | jq -r '.assets[].browser_download_url' | grep << parameters.platform >>-<< parameters.architecture >>)
+        echo $asset_url
```

Comment:
debugging line got left behind?
```suggestion
```

Reply:
That was intentional so that you'd see the URL to sanity check, but I now see that it's already visible in the job output without this line so I'll remove 👍🏼